### PR TITLE
Fixing lots of little issues, removing closed programs

### PR DESCRIPTION
**Pull request by karllhughes**
- Removed closed programs
- Fixed some typos and out of order entries
- Updated some pay rates
- Added details to a few programs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,34 +67,31 @@ A list of companies that have paid Developer Community Writer Programs.
...
-- [Digital Ocean](https://www.digitalocean.com/write-for-donations/) - Up to $400 per piece **Applications currently on hold, please check back on the first of March 2022!**
+- [Digital Ocean](https://www.digitalocean.com/community/pages/write-for-digitalocean) - Up to $400 per piece
   > Technical tutorials with code. Not limited to Digital Ocean products.
 
 - [Dockship](https://dockship.io/articles) - $20
   > Machine Learning and Data Science. You need to be signed in to be able to create content.
 
-- [Doppler](https://www.doppler.com/tech-writer-program) - $100 per article
-  > Seeking deep-dive technical tutorials focussed on application security, secrets management, and developer tools.
-
```
Comment (karllhughes) [inline README.md:84]: Page is dead, so assuming this program is gone.

File: README.md [hunk 2]
```diff
@@ -67,34 +67,31 @@ A list of companies that have paid Developer Community Writer Programs.
...
 - [Draft.dev](https://draft.dev/write) - Pays $315-$578 per piece
   > Technical content agency that works with many clients. Writers who are accepted will get an email every week and access to a writer portal with dozens of topics they can choose from.
 
-- [DZone](https://dzone.com/writers-zone) - Waiting for details
-  > Broad coverage of development topics, but with heavy Java content, working hard to push into new topics and channels.
```
Comment (karllhughes) [inline README.md:89]: DZone doesn't pay, so I'm not sure why this is on there.

File: README.md [hunk 3]
```diff
@@ -67,34 +67,31 @@ A list of companies that have paid Developer Community Writer Programs.
...
-
-- [Egghead](https://next.egghead.io/write-for-egghead) - Waiting for details
+- [Egghead](https://next.egghead.io/write-for-egghead) - Pay unknown
   > Intermediate to advanced articles covering topics on web development.
 
-- [Fauna](https://fauna.com/blog/write-with-fauna) - Up to $350 per piece **Applications currently on hold**
+- [Enlear](https://www.enlear.com/) - $50 to $150 per article
+  > Get paid to write about broad area of technical topics. Including Vue, Angular, React, Node, JS, DynamoDB
+
+- [Fauna](https://fauna.com/blog/write-with-fauna) - Up to $350 per piece
```
Comment (karllhughes) [inline README.md:91]: Fauna is back

File: README.md [hunk 4]
```diff
@@ -109,10 +106,7 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Honeybadger](https://www.honeybadger.io/blog/write-for-us/) - From $500 per piece
   > Ruby and Elixir tutorials with code. Pick from a list of possible articles.
 
-- [ImageKit](https://form.asana.com/?hash=4a457043221411a8d81971bc930a6a932add03c8eaeec167cf1fee81db871953&id=1166170812623393) - $300 per piece
-  >  Content focused on technical guide on ImageKit and Image Optimization.
-
-- [Linode](https://www.linode.com/docs/contribute/) - Up to $300 per piece
+- [Linode](https://www.linode.com/lp/write-for-linode/) - Up to $400 per piece
```
Comment (karllhughes) [inline README.md:109]: Updated their pay rate

File: README.md [hunk 5]
```diff
@@ -125,23 +119,17 @@ A list of companies that have paid Developer Community Writer Programs.
   > Technical content and tutorials about DevSecOps, Cloud security, Kubernetes security.
 
 - [Magic](https://magic-fortmatic.typeform.com/to/Wgzsocor) - Up to $300 per piece
-  [Link to Magic Website](https://magic.link/)
-  > Technical tutorials on how to use Magic
+  > Technical tutorials on how to use Magic.link
```
Comment (karllhughes) [inline README.md:122]: Just added their website to the description

File: README.md [hunk 6]
```diff
@@ -125,23 +119,17 @@ A list of companies that have paid Developer Community Writer Programs.
...
 
 - [Make Use Of](https://www.makeuseof.com/contributor/) - $120 per piece with performance benefits
   > Tutorials and features about consumer apps and software products.
 
-- [Mixster](https://mixstersite.wordpress.com/2019/05/24/mixster/#more-2253)
```
Comment (karllhughes) [inline README.md:134]: This one is really confusing. It seems like maybe they're looking for writers, but there's just a Twitter thread, not a clear way to apply.

File: README.md [hunk 7]
```diff
@@ -125,23 +119,17 @@ A list of companies that have paid Developer Community Writer Programs.
...
-
-- [Nanonet](https://nanonets.com/blog/write-for-us/)
-  > Get paid to write about your favourite machine learning topics!
+- [Nanonet](https://nanonets.com/blog/write-for-us/) - Pay unknown
+  > Get paid to write about your favorite machine learning topics.
 
-- [Neptune](https://neptune.ai/write-for-us) - Up to $500 per article.
+- [Neptune](https://neptune.ai/write-for-us) - $300-$600 per article.
```
Comment (karllhughes) [inline README.md:130]: New pay rate

File: README.md [hunk 8]
```diff
@@ -125,23 +119,17 @@ A list of companies that have paid Developer Community Writer Programs.
...
   > Technical articles, how-to guides and tutorials on machine learning and data science.
 
-- [Okta](https://developer.okta.com/blog) - Paid through Toptal based on your hourly rate
-  > Technical tutorials and demos using Okta's products.
```
Comment (karllhughes) [inline README.md:143]: This one is confusing. You can't directly apply to work with Okta, you have to go through Toptal, so I removed this.

File: README.md [hunk 9]
```diff
@@ -154,24 +142,12 @@ A list of companies that have paid Developer Community Writer Programs.
...
   > Thought leadership and technical articles about PHP.
 
-- [Postmark](https://postmarkapp.com/write-for-us) - $200-$300 per piece
-  > Applications are currently closed.
-
 - [QuickNode](https://quicknode.notion.site/quicknode/QuickNode-Authorship-Program-d808a87ee50b48c9a16ed19b13e09115) - $350 per piece
   > Get paid to write articles about cryptocurrencies and web3/blockchain.
 
-- [Rancher](https://rancher.com/writing-program/roles/writer/) - $300 per piece
-  > Writing about devops, Kubernetes, and Rancher.
```
Comment (karllhughes) [inline README.md:164]: Page 404'd.

I know Rancher got bought by Suse last year, so probably shut this down.

File: README.md [hunk 10]
```diff
@@ -154,24 +142,12 @@ A list of companies that have paid Developer Community Writer Programs.
...
-
-- [Real Python](https://realpython.com/write-for-us/) - **Applications currently on hold** 
-  > Technical tutorials with code. Pick from a list of possible articles.
```
Comment (karllhughes) [inline README.md:167]: Apps closed, no idea when they'll re-open, so removing this.

File: README.md [hunk 11]
```diff
@@ -154,24 +142,12 @@ A list of companies that have paid Developer Community Writer Programs.
...
-
 - [Sanity.io](https://www.sanity.io/guest-authorship) - Up to $250 per piece
   > Technical focused articles and how-to guides. Pick from a list of possible articles.
 
-- [Section.io](https://github.com/section-io/engineering-education) - $50 to $150 per article
-  > Engineering education blog where Computer Science university students may contribute content for pay.
```
Comment (karllhughes) [inline README.md:173]: They no longer pay, so I'm removing this.

File: README.md [hunk 12]
```diff
@@ -181,12 +157,13 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Smashing Magazine](https://www.smashingmagazine.com/write-for-us/) - $200 to $250 per article
   > Technical focused articles. No limitation on topics.
 
-- [Software Engineering Daily](https://softwareengineeringdaily.com/write/)
+- [Software Engineering Daily](https://softwareengineeringdaily.com/write/) - Pay unknown
+  > We explain how software is built. We explain how software has gotten us to where we are, and how new software will shape our future.
 
-- [Soshace](https://docs.google.com/document/d/1DZ9Hj8AcNfHI6bC4bfTDIFRNIIFnda6Mkj_n_4x3hWw/edit) - $100 per piece
+- [Soshace](https://blog.soshace.com/write-for-us/) - $100 per piece
   > Technical tutorials with code. Pick from a list of possible articles.
```
Comment (karllhughes) [inline README.md:164]: New link to apply

File: README.md [hunk 13]
```diff
@@ -201,9 +178,6 @@ A list of companies that have paid Developer Community Writer Programs.
 - [TechWell](https://www.techwell.com/techwell-submission-guidelines) - $200 per piece
   > A wide variety of technical and business content is considered.
 
-- [TestDriven.io](https://testdriven.io/blog/) - $300-$500 per guest post
-  > Web development tutorials designed to teach critical skills needed to test, launch, scale, and optimize applications.
```
Comment (karllhughes) [inline README.md:205]: No application form or info about how to apply, so I'm removing this.

File: README.md [hunk 14]
```diff
@@ -216,20 +190,14 @@ A list of companies that have paid Developer Community Writer Programs.
 - [TypingDNA](https://www.typingdna.com/guest-author-program) - Up to $500 per piece
   > Technical articles/tutorials related to TypingDNA.
 
-- [Vonage](https://developer.nexmo.com/spotlight/) - $500 per piece
+- [Vonage](https://learn.vonage.com/spotlight/) - $500 per piece
```
Comment (karllhughes) [inline README.md:193]: New link

File: README.md [hunk 15]
```diff
@@ -216,20 +190,14 @@ A list of companies that have paid Developer Community Writer Programs.
...
   > Technical tutorials and general pieces on programming
 
 - [Vultr](https://www.vultr.com/docs/vultr-docs-program-guidelines) - Up to $600 per piece
   > most topic take a look on Vultr document
 
-- [Webapp.io](https://webapp.io/technical-writer-program) - $150+ per piece
-  > Technical content focused on WebApp projects, Docker, Node and Ruby.
```
Comment (karllhughes) [inline README.md:226]: Page not found, assuming this program is no more.

File: README.md [hunk 16]
```diff
@@ -216,20 +190,14 @@ A list of companies that have paid Developer Community Writer Programs.
...
-
-- [WPHUB](https://www.wphub.com/write-for-us/) - $200 per piece
+- [WPHUB](https://www.wphub.com/write-for-us/) - $100-$200 per piece
```
Comment (karllhughes) [inline README.md:199]: Updated pricing.

File: README.md [hunk 17]
```diff
@@ -52,8 +52,8 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Code Tuts+](https://code.tutsplus.com/articles/call-for-authors-write-for-tuts--cms-22034) - $100 (Quick tip) $250 (Tutorial)
   > Technical focused articles. Pick from a list of possible articles.
 
-- [Code Magic](https://blog.codemagic.io/write-for-codemagic-ci-cd/) - Applications are currently closed.
-  > Technical focused articles. Specific on Flutter.
+- [Code Magic](https://blog.codemagic.io/write-for-codemagic-ci-cd/) - Pay unknown
+  > Technical articles about Flutter.
```
Comment (karllhughes) [inline README.md:56]: Apps appear to be open again, but pay is unknown.

**Comment by karllhughes** [comment]
@malgamves @ChrisChinchilla time for some spring cleaning! 🧹

I did a big round of cleanup on our list this morning. I left comments on specific updates in this PR, so please let me know if you have any objections, otherwise, I'll plan on merging this in at the end of next week. Thanks!

File: README.md (hunk 17 above)
Comment (xTrilton) [reply README.md:56]: Hello, they recruit writers who have posted articles on Medium that talk about codeMagic.

Comment (karllhughes) [reply README.md:56]: Ok, do you know how much they pay? I can add more details if you have any.

**Comment by codeInn001** [comment]
This is nice. Personally, I don't think you should remove okta. Great work @karllhughes

**Comment by karllhughes** [comment]
> This is nice. Personally, I don't think you should remove okta. Great work @karllhughes

Yeah, I was on the fence there, so thanks for the input. Added it back as there's no harm keeping it on there.

File: README.md (hunk 6 above)
Comment (malgamves) [reply README.md:137]: true, might be worth removing for now

**Review by malgamves — APPROVED** [review]

**Comment by codeInn001** [comment]
> > This is nice. Personally, I don't think you should remove okta. Great work @karllhughes
> 
> Yeah, I was on the fence there, so thanks for the input. Added it back as there's no harm keeping it on there.

Exactly and you've alrea

> > This is nice. Personally, I don't think you should remove okta. Great work @karllhughes
> 
> Yeah, I was on the fence there, so thanks for the input. Added it back as there's no harm keeping it on there.

Yeah. Also the description also shows paid through toptal.

**Comment by codeInn001** [comment]
> > This is nice. Personally, I don't think you should remove okta. Great work @karllhughes
> 
> Yeah, I was on the fence there, so thanks for the input. Added it back as there's no harm keeping it on there.

Exactly and the description shows paid through toptal.

**Pull request merged**